src/EventGridExtension/EventGridExtensionConfig.cs

```diff
@@ -52,7 +52,7 @@ internal void AddListener(string key, EventGridListener listener)
             _listeners.Add(key, listener);
         }
 
-        async Task<HttpResponseMessage> IAsyncConverter<HttpRequestMessage, HttpResponseMessage>.ConvertAsync(HttpRequestMessage input, CancellationToken cancellationToken)
+        public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, CancellationToken cancellationToken)
```

Comment:
Does this need to be public? If you changed it scope for testing purposes, then would internal be enough?

Reply:
https://github.com/Azure/azure-functions-eventgrid-extension/blob/d92a85a026698ac3a759de974cab35f6f73a0991/src/EventGridExtension/EventGridExtensionConfig.cs#L17-L18
This is an implementation of ` IAsyncConverter` interface, which is supposed to be public. I was using [explicit implementation](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation) but I can no longer justify it since that's usually applied when multiple interfaces have members of the same signature

Reply:
its not just used in my test, its also called by [function host](https://github.com/Azure/azure-functions-host/blob/dc87f344e30b94b2a36ff9afe61c51af57388ad9/src/WebJobs.Script.WebHost/Controllers/AdminController.cs#L205-L227)
```
        public async Task<HttpResponseMessage> ExtensionWebHookHandler(string name, CancellationToken token)
        {
            var provider = _scriptHostManager.BindingWebHookProvider;

            var handler = provider.GetHandlerOrNull(name);
            if (handler != null)
            {
                string keyName = WebJobsSdkExtensionHookProvider.GetKeyName(name);
                if (!this.Request.HasAuthorizationLevel(AuthorizationLevel.System, keyName))
                {
                    return new HttpResponseMessage(HttpStatusCode.Unauthorized);
                }

                return await handler.ConvertAsync(this.Request, token); <---here
            }

            return new HttpResponseMessage(HttpStatusCode.NotFound);
        }
```